openshift-with-appstudio-test/e2e/client.go

```diff
@@ -10,6 +10,8 @@ import (
 	projectset "github.com/openshift/client-go/project/clientset/versioned"
 	jvmclientset "github.com/redhat-appstudio/jvm-build-service/pkg/client/clientset/versioned"
 	pipelineclientset "github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
I would prefer this k8s import be coupled with the other k8s imports below (i.e right before the `kubeset` import )